Woo 2.0 Brand Updates

irfano opened the pull request:
<!-- Remember about a good descriptive title. -->
Part of: woocommerce/woomobile-private#373

### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
This PR merges the main feature branch of the Brand Updates project (pffQ75-10m-p2) into `trunk`. It updates the color theme, logo, and illustrations.

This PR only includes merge commits from other project PRs, syncing with `trunk` commits, and a commit for updating RELEASE-NOTES.

### Steps to reproduce
<!-- Step-by-step testing instructions. For new user flows, consider instead stating the goal of the workflow and see if your PR reviewer can accomplish the workflow without specific steps! -->
All changes have already been tested in individual PRs. For this PR, a general test should ve sufficient. You can follow the test plan (pe5sF9-3vc-p2). Items labeled "**Extra cases:**" are optional to test.

### Testing information
<!-- This is your opportunity to break out individual scenarios that need testing (when necessary) and/or include a checklist for the reviewer to go through. Consider documenting the following from your own completed testing: devices used, alternate workflows, edge cases, affected areas, critical flows, areas not tested, and any remaining unknowns. Provide feedback on this new section of the PR template through Sept 30, 2024 to Apps Quality; additional context here: p91TBi-b8z-p2#comment-12036 -->
Please test both the light and dark modes.

### The tests that have been performed
<!-- To give the reviewer an idea of what could be missed in terms of testing -->
All the commits here have already been tested in other PRs.
Cases in the test plan.

### Images/gif
<!-- Include before and after images or gifs when appropriate. -->
|Before|After|
|-|-|
|<img src="https://github.com/user-attachments/assets/9fa0ff2f-8eb2-48a4-acc4-f7d483a7671c" width=480>|<img src="https://github.com/user-attachments/assets/21a11c6e-d64a-4572-9200-cce255115495" width=480>|

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

irfano committed:
Remove unused colors

irfano committed:
Remove unused colors

irfano committed:
Replace hardcoded colors with const colors in WooPosTheme

irfano committed:
Replace hardcoded colors with const colors in WooPosTheme

irfano committed:
Update Woo purples in xml files

irfano committed:
Update Woo purples in xml files

irfano committed:
Update Woo color reference link

irfano committed:
Update Woo color reference link

irfano committed:
Update Woo colors in WooPosTheme

irfano committed:
Update Woo colors in WooPosTheme

irfano committed:
Update Woo purples in WooColors

irfano committed:
Update Woo purples in WooColors

irfano committed:
Remove Purple colors from WooPostheme to use WooColors

Removed purple colors were already Woo Purple colors in color-studio.blog. There's no need to keep purple and WooPurple.

irfano committed:
Remove Purple colors from WooPostheme to use WooColors

Removed purple colors were already Woo Purple colors in color-studio.blog. There's no need to keep purple and WooPurple.

irfano committed:
Update Woo colors in FeedbackDialog

irfano committed:
Update Woo colors in FeedbackDialog

irfano committed:
Update ic_launcher_background color

irfano committed:
Update ic_launcher_background color

irfano committed:
Update splash screen background color

irfano committed:
Update splash screen background color

irfano committed:
Update the colorPrimary of the theme

irfano committed:
Update the colorPrimary of the theme

irfano committed:
Remove alpha from BottomNavigationView icons

irfano committed:
Remove alpha from BottomNavigationView icons

irfano committed:
Remove unused drawable

irfano committed:
Remove unused drawable

irfano committed:
Add style to a button in the product filter

irfano committed:
Add style to a button in the product filter

irfano committed:
Merge pull request #13147 from woocommerce/feature/update-theme-colors

Update theme colors

irfano committed:
Update playstore icon

irfano committed:
Delete ic_launcher_foreground.png files

These images are not necessary.

irfano committed:
Remove mipmap-anydpi-v26 files for wasabi and jalapeno

These are removed because they're the same for all build variants.

irfano committed:
Remove -v26 suffix from mipmap-anydpi-v26

Our minSdk is 36 now, so we don't need -c26 suffix anymore.

irfano committed:
Update ic_launcher_foreground.xml icon

irfano committed:
Update png versions of ic_launcher and ic_launcher_round icons

irfano committed:
Update ic_launcher icons for the Wear app

irfano committed:
Update Woo logo used in the app

irfano committed:
Set the primary color to Purple 30 for dark mode

irfano committed:
Align the WooPosTheme colors with the app’s colors

irfano committed:
Update the primary color for wear

irfano committed:
Update ic_woo_bubble

irfano committed:
Update Jetpack connection illustrations

irfano committed:
Update ic_woo_illustrated_icon

irfano committed:
Update TTP illustrations

irfano committed:
Merge pull request #13185 from woocommerce/feature/update-primary-colors

Update primary colors

hafizrahman committed:
Merge pull request #13186 from woocommerce/feature/update-images-in-jc-flow

Update illustrations in the JC flow

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/update-app-icons

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/update-woo-logo-iin-the-app

irfano committed:
Fix issue on card reader connect dialog when header_label is too long

irfano committed:
Update reader illustrations

irfano committed:
Update empty order illustration

irfano committed:
Update share store illustration

irfano committed:
Update empty products illustration

irfano committed:
Update empty coupons illustration

irfano committed:
Update empty reviews illustration

irfano committed:
Update empty search illustration

irfano committed:
Update empty grouped product illustration

irfano committed:
Update dashboard card error illustration

irfano committed:
Revert primary color change

irfano committed:
Revert "Revert primary color change"

This reverts commit 2efef39afaecc67899a78ab58ae1b4344fad6a63.

irfano committed:
Revert primary color change for night

irfano committed:
Update the Woo logo on the splash screen for sasabi

irfano committed:
Remove unnecessary woo logo for jalapeno variant

irfano committed:
Rename img_woo_bubble and remove bubble from it

hafizrahman committed:
Merge pull request #13204 from woocommerce/feature/update-reader-illustrations

Update reader illustrations

irfano committed:
Merge pull request #13169 from woocommerce/feature/update-woo-logo-iin-the-app

Update Woo logo used in the app

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/update-app-icons

irfano committed:
Add `-v26` suffix back to `mipmap-anydpi` directory

irfano committed:
Update the Woo logo on the splash screen for the Wasabi variant

irfano committed:
Merge pull request #13166 from woocommerce/feature/update-app-icons

Update app icons

hafizrahman committed:
Merge pull request #13194 from woocommerce/feature/update-ttp-illustrations

Update TTP illustrations

irfano committed:
Enable edge to edge for LoginActivity

irfano committed:
Update prologue carousel background image

irfano committed:
Update Woo logo on prologue screens

irfano committed:
Make skip button on prologue carousel screens black

irfano committed:
Update prologue carousel screen layout

irfano committed:
Update prologue carousel item layout for landscape and tablets

irfano committed:
Update prologue carousel item layout

irfano committed:
Update prologue carousel illustrations

irfano committed:
Rename PrologueFinishedListener to PrologueListener

An additional function, unrelated to finishing, will be added to the interface in the next commit.

irfano committed:
Add margin to the prologue skip button based on the navigation bar

irfano committed:
Handle window insets at the start and end of the prologue screens

irfano committed:
Update the primary color for dark mode

The primary color should be set to Woo 30 for dark mode and implemented in #13185.

irfano committed:
Update indicator colors

irfano committed:
Rename ic_tab_indicator.xml to ic_pager_indicator.xml

irfano committed:
Merge pull request #13208 from woocommerce/feature/update-empty-illustrations

Update empty illustrations

irfano committed:
Add a comment explaining the usage of the enableEdgeToEdge() function

irfano committed:
Merge pull request #13234 from woocommerce/feature/update-prologue-carousel-screens

Update prologue carousel screens

irfano committed:
Rename img_prologue_bg to img_prologue_bg_white

irfano committed:
Update img_prologue.webp drawable

irfano committed:
Update login prologue screen layouts

irfano committed:
Update img_prologue_bg_white usage in prlogue carousel screens

irfano committed:
Add DynamicEdgeToEdgeActivity interface

irfano committed:
Handle system bar instets on the login prologue screen

irfano committed:
Implement DynamicEdgeToEdgeActivity in LoginActivity

irfano committed:
Remove unnecessary resources

irfano committed:
Update the tap card illustration

irfano committed:
Update img_woo_payments_logo.xml

irfano committed:
Update img_woo_payments.xml

irfano committed:
Update notification icon

irfano committed:
Update login notification icon

irfano committed:
Update the background color for login notification

irfano committed:
Update bell icon

irfano committed:
Update timer icon

irfano committed:
Remove unused icons

irfano committed:
Update purple colors in icons

t

irfano committed:
Fix the typo in checkbox drawable name

irfano committed:
Update not found icon

Renamed the icon and added its dark version.

irfano committed:
Update top performers empty card image

irfano committed:
Update widget preview

hafizrahman committed:
Merge pull request #13238 from woocommerce/feature/update-tap-card-illustration

Update the tap card illustration

hafizrahman committed:
Merge pull request #13239 from woocommerce/feature/update-woo-payments-logo

Update woo payments logo

hafizrahman committed:
Merge pull request #13241 from woocommerce/feature/update-illustrated-icons

Update illustrated icons

hafizrahman committed:
Merge pull request #13243 from woocommerce/feature/update-top-performers-empty-image

Update top performers card empty image

hafizrahman committed:
Merge pull request #13244 from woocommerce/feature/update-widget-review

Update widget preview

hafizrahman committed:
Merge pull request #13237 from woocommerce/feature/update-prologue-login-screen-design

Update prologue login screen design

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/update-notification-icon

hafizrahman committed:
Merge pull request #13242 from woocommerce/feature/update-icon-colors

Update icon colors

irfano committed:
Merge pull request #13240 from woocommerce/feature/update-notification-icon

Update notification icon

irfano committed:
Update generic error illustration

hafizrahman committed:
Merge pull request #13248 from woocommerce/feature/update-generic-error-illustration

Update generic error illustration

irfano committed:
Update find your store address popup illustration

irfano committed:
Update dark version of ic_not_found icon

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

dangermattic commented:
<!--
  0 Errors
  2 Warnings: This PR is larger than 300 lin...
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/249">21.6</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

hafizrahman committed:
Merge pull request #13252 from woocommerce/feature/update-find-your-store-address-popup

Update find your store address popup illustration

irfano committed:
Remove unused Woo logos from Wear

irfano committed:
Update Woo loading logo for Wear

irfano committed:
Update monochrome Wear logo

irfano committed:
Update About library version

irfano committed:
Merge pull request #13253 from woocommerce/feature/update-not-found-icon-color

Update dark version of ic_not_found icon

hafizrahman committed:
Merge pull request #13263 from woocommerce/feature/update-about-library

Update About library version

irfano committed:
Update rounding of reader illustrations

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

hafizrahman committed:
Merge pull request #13280 from woocommerce/feature/update-roundings-of-reader-illustrations

Update radius of reader illustrations

irfano committed:
Fix disabling edge-to-edge for lower APIs

irfano committed:
Fix enabling edge-to-edge for lower APIs on carousel screen

hafizrahman committed:
Merge pull request #13262 from woocommerce/feature/update-wear-logos

Update wear logos

irfano committed:
Merge pull request #13287 from woocommerce/fix/navigation-bar-issue-on-carousel-screen

Navigation bar issue on prologue screens for low APIs

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Change ic_woo_pos_empty_products.xml icon with ic_not_found.xml

irfano committed:
Add missing periods to sentences

irfano committed:
Remove unused `androidx-activity` line from libs.versions.toml

irfano committed:
Update wordpress-login library reference

irfano committed:
Merge pull request #13352 from woocommerce/feature/add-missing-periods

Add missing periods

irfano committed:
Remove underline from html links

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Rename app to Woo

irfano committed:
Remove unused drawables

irfano committed:
Update Wear app logos for Wasabi

irfano committed:
Update jalapeno app icon

irfano committed:
Remove png and webp app icons

Since our minSdk is now 26, we no longer need these image files.

irfano committed:
Add more missing periods

irfano committed:
Merge pull request #13361 from woocommerce/feature/rename-app-to-woo

Rename app to Woo

irfano committed:
Update the link color on free domain registration screen

irfano committed:
Update login flow text button colors

irfano committed:
Update login-flow library to the PR reference

irfano committed:
Update neutral button color of login dialogs

irfano committed:
Update button color of find your site address dialog in site picker

irfano committed:
Remove strings to resolve "Inconsistent number of arguments" warning

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Update the login prologue screen layout to match the previous screen

irfano committed:
Improve text wrapping in Login Prologue screens

Adds `android:breakStrategy="balanced"` to TextViews in the Login Prologue screens to improve text wrapping and avoid awkward line breaks.

hafizrahman committed:
Merge pull request #13362 from woocommerce/feature/update-variant-app-logos

Update variant app logos for Wear

hafizrahman committed:
Merge pull request #13363 from woocommerce/feature/add-more-missing-periods

Add more missing periods

irfano committed:
Merge pull request #13380 from woocommerce/feature/update-login-prologue-screen-layout

Update the login prologue screen layout to match the previous screen

irfano committed:
Update img_install_jetpack.png for dark mode

irfano committed:
Update img_connect_jetpack.png for dark mode

irfano committed:
Update img_jetpack_connection_dismissed.png for dark mode

irfano committed:
Update img_site_picker_no_stores.webp.png

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Merge pull request #13392 from woocommerce/feature/update-dark-mode-illustrations

Update dark mode illustrations

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/remove-underline-from-links

# Conflicts:
#	WooCommerce/src/main/res/values-es/strings.xml
#	WooCommerce/src/main/res/values-id/strings.xml
#	gradle/libs.versions.toml

irfano committed:
Update wordpress-login library

irfano committed:
Merge branch 'feature/woo-2.0-brand-updates' into feature/update-text-button-colors-in-dark-mode

# Conflicts:
#	gradle/libs.versions.toml

irfano committed:
Update wordpress-login library

irfano committed:
Merge pull request #13360 from woocommerce/feature/remove-underline-from-links

Remove underline from links

irfano committed:
Merge pull request #13377 from woocommerce/feature/update-text-button-colors-in-dark-mode

Update the button colors in the login flow

irfano committed:
Remove unused string of "Twitter"

irfano committed:
Update About library

irfano committed:
Merge pull request #13412 from woocommerce/feature/update-about-library

Update the About library

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates

irfano committed:
Update RELEASE-NOTES.txt

irfano committed:
Update line height for prologue headers

JorgeMucientes committed:
Merge pull request #13420 from woocommerce/feature/update-login-prologue-headers-size

Update line height for prologue headers

irfano committed:
Update `ic_launcher_foreground.xml` scale

The scale values for `ic_launcher_foreground.xml` have been updated from 0.7213954 to 0.61.

JorgeMucientes commented:
Awesome job in this whole project @irfano :kudos 🙌🏼 

I smoke-tested the app's main flows as well as some optional ones. I found a few minor things that might be worth checking because I'm not sure if their are bugs or the color usage is intentional. In any case, none are blocking imho:

#### Filters with old pink. 
<img width="300"  src="https://github.com/user-attachments/assets/8702878a-6ba6-4f30-983f-a5a6a13fd1f7"> 
<img width="300"  src="https://github.com/user-attachments/assets/12a826af-9172-4fc9-9f39-8349068c3edb"> 


#### Close Account pink color
<img width="300"  src="https://github.com/user-attachments/assets/3e128f04-ad64-441a-87e2-ece499a9f3a3"> 

#### Badge pink color
<img width="300"  src="https://github.com/user-attachments/assets/1463d09c-68fa-4da0-a23f-967e255d13b3">

irfano commented:
Thanks for reviewing, @JorgeMucientes! These pink colors haven't changed, and they use the latest pink tones from [Color Studio](https://color-studio.blog/).

irfano committed:
Update app icon size for the wasabi variant

irfano committed:
Merge pull request #13422 from woocommerce/feature/udpate-app-logo-size

Update app logo size

irfano committed:
Merge branch 'trunk' into feature/woo-2.0-brand-updates
